### PR TITLE
[Artur Kostiuk] First_layout

### PR DESCRIPTION
**Pull request by art1kos**

### CODE REVIEW

**Comment by tarasbobak** [comment]
@art1kos Please deploy it to github pages.

File: css-practice/task_1/index.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+   <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.2/css/all.css" integrity="sha384-/rXc/GQVaYpyDdyxK+ecHPVYJSN9bmVFBvjA/9eOB+pb3F2w2N6fc5qB9Ew5yIns" crossorigin="anonymous">
+      <link rel="stylesheet" href="style/main.css">
+      <title>MiNiMo</title>
+   </head>
+   <body>
+      <header id = "main-header">
```
Comment (tarasbobak) [inline css-practice/task_1/index.html]: id="main-header"

Comment (tarasbobak) [reply css-practice/task_1/index.html]: Why do you use id's?
Change it to classes whenever possible

File: css-practice/task_1/index.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+            <div class="menu">
+               <ul>
+                  <li><a href="#" class="menu-link">lifestyle</a></li>
+                  <li><a href="#" class="menu-link">photodiary</a></li>
+                  <li><a href="#" class="menu-link">music</a></li>
+                  <li><a href="#" class="menu-link">travel</a></li>
+               </ul>
+            </div>
+         </div>
+         <div class="baner">
```
Comment (tarasbobak) [inline css-practice/task_1/index.html]: Do you think banner is a part of header?

File: css-practice/task_1/index.html [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+      <title>MiNiMo</title>
+   </head>
+   <body>
+      <header id = "main-header">
+        <div class="container">
+          <div class="header">
+            <div class="logo">
+               <a href="#" class="menu-link"><img src="img/minimø.png" alt="logo"></a>
+            </div>
+            <div class="menu">
```
Comment (tarasbobak) [inline css-practice/task_1/index.html]: how about nav element?

File: css-practice/task_1/index.html [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+                  <li><a href="#" class="menu-link">travel</a></li>
+               </ul>
+            </div>
+         </div>
+         <div class="baner">
+            <img src="img/image.png" alt="main-img">
+         </div>
+        </div>
+      </header>
+      <div class="content">
```
Comment (tarasbobak) [inline css-practice/task_1/index.html]: Why you don't use HTML 5 semantic elements?

File: css-practice/task_1/index.html [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+         <div class="photodiary">
+            <span>photodiary</span>
+            <h2>The perfect weekend getaway</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit,
+               sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+               Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
```
Comment (tarasbobak) [inline css-practice/task_1/index.html]: Why is each sentence on new line?

File: css-practice/task_1/index.html [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+               Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
+               Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. 
+               Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, 
+               eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam 
+               voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem 
+               sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non 
+               numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem.
+            </p>
+         </div>
+         <div class="card-holder">
```
Comment (tarasbobak) [inline css-practice/task_1/index.html]: I would do this with flex or inline block and single container

File: css-practice/task_1/style/main.css [hunk 7]
```diff
@@ -0,0 +1,712 @@
...
+  .footer-menu{
+    width: 40%
+  }
+  .footer-menu li a{
+    font-size: 12px;
+  } 
+  .follow-link p{
+    font-size: 12px;
+  }
+  .fa-facebook-f{
```
Comment (tarasbobak) [inline css-practice/task_1/style/main.css]: can all of these be listed and separated by comma?

**Comment by art1kos** [comment]
https://art1kos.github.io/ui-internship-lwo2/css-practice/task_1/index.html#

**Pull request merged**